src/io.h

```diff
@@ -26,6 +26,26 @@ bool writeFile(const std::string &file, const std::string &s,
 
 void strip(std::vector<std::string> &string_vector, const std::string &pattern); //!< Strip lines matching a pattern
 
+/**
+ * Write a map to a file as key-value pairs
+ * @tparam TKey
+ * @tparam TValue
+ * @param filename
+ * @param data
+ */
+template<typename TKey, typename TValue>
+void write(const std::string &filename, const std::map<TKey, TValue>& data) {
```

Comment:
Make an overload `write(std::ostring &, const std::map<> &)` to enable streaming to memory or compressed streams. Useful for unit testing. Perhaps rename to `writeMap`?

Reply:
sorry `std::ostream`

Reply:
It is not needed. To keep pragmatically the code base small, I won’t fix it